source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -30,7 +30,7 @@ content: |
         .. code-block:: javascript
            :emphasize-lines: 5
 
-           const path = "./master-key.txt";
+           const path = './master-key.txt';
            let localMasterKey;
            fs.readFile(path, (err, data) => {
```

Comment:
This still needs to be changed to a synchronous version:

```js
const path = './master-key.txt';
const localMasterKey = fs.readFileSync(path);
```

Reply:
Missed this one, thanks for spotting it. Will fix.